generator/plugins/swagger2gql/services.go

```diff
@@ -147,7 +147,13 @@ func (p *Plugin) tagMutationsMethods(tagCfg TagConfig, file *parsedFile, tag par
 func (p *Plugin) fileServices(file *parsedFile) ([]graphql.Service, error) {
 	var res []graphql.Service
 	for _, tag := range file.File.Tags {
-		tagCfg := file.Config.Tags[tag.Name]
+		tagCfg, ok := file.Config.Tags[tag.Name]
+
+		if !ok {
+			fmt.Println("Warning: Skip tag:", tag.Name, "from", file.OutputPath)
```

Comment:
why you here print `file.OutputPath`, maybe we need to log here path to swaggerfile or maybe `file.Name`?

Reply:
Done, thanks.